src/screens/CodeVerifyScreen/CodeVerify.tsx

```diff
@@ -3,7 +3,6 @@ import React, {useRef, useState} from "react";
 import {useSelector} from "react-redux";
 import {useNavigation} from "@react-navigation/native";
 import {styles} from "./style";
-import Animated from "react-native-reanimated";
 import {displayErrorMessage} from "../../utils/displayMessage";
 import {RootState} from "../../redux/reducers/rootReducer";
 import {AnimatedBackground} from "./AnimatedBackground";
```

Comment:
`const {code} = useSelector((state: RootState) => state.user);`
This is bad for performance in various respects. We should create appropriate selectors for every field we use in code. We shouldn't pass an arrow function as an argument to useSelector. See https://redux.js.org/tutorials/fundamentals/part-5-ui-react#reading-state-from-the-store-with-useselector